test/cljs/status_im/test/i18n.clj

```diff
@@ -0,0 +1,9 @@
+(ns status-im.test.i18n)
+
+;; translator: add your language to this list
+(def languages-to-test ["ko"])
+
+(defmacro translations []
```

Comment:
Why does this need to be a macro? Isn't it just data?

Reply:
The translations are maps that are named 'language/translations', in order to get this symbol from the language string such as "ko" or "fr" we need to construct the symbol 'ko/translations', 'fr/translations' and you can only do this in a macro afaik.

Reply:
You don't need symbol, you can just import and use keys directly?